src/Inciter/Carrier.h

```diff
@@ -106,6 +107,7 @@ namespace inciter {
 
 extern ctr::InputDeck g_inputdeck;
 extern CkReduction::reducerType VerifyBCMerger;
+extern CkReduction::reducerType PDFMerger;
```

Comment:
AFAIK, this must be global. See the comments starting at https://github.com/quinoacomputing/quinoa/blob/master/src/Inciter/Carrier.h#L161.